homeassistant/const.py

```diff
@@ -118,6 +118,7 @@
 CONF_PREFIX = "prefix"
 CONF_PROFILE_NAME = "profile_name"
 CONF_PROTOCOL = "protocol"
+CONF_PROVIDER = "provider"
```

Comment:
Please define it in your own integration if you are the only user for this const.

Reply:
actually gpslogger uses `ATTR_PROVIDER` with the same string. 
I'll define my own const for now.